### PR TITLE
Bookmark page

### PR DESCRIPTION
**Pull request by Dariastep**

### CODE REVIEW

File: bookmarks.html [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Quiz-App</h1>
+    </header>
+    <main>
+      <ul class="card-list">
+        <li class="question-card">
+          <a href="#"
```
Comment (julialoeschel) [inline bookmarks.html:17]: you migth want to think about aria - label here.

File: bookmarks.html [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+          <button type="button" class="answer-button">
+            <p class="button-text">Show Answer</p>
+          </button>
+          <ul class="tag-box-list">
+            <li class="tag-box-item">#html</li>
+            <li class="tag-box-item">#flexbox</li>
+            <li class="tag-box-item">#css</li>
+          </ul>
+        </li>
+      </ul>
```
Comment (julialoeschel) [inline bookmarks.html:49]: you are using more `ul `s than necesssary here

you can put more `li`in one `ul`

**Review by julialoeschel — COMMENTED** [review]
🦅 
great work. 

I left some comments for you to consider in future code :) 
keep up the good work

File: bookmarks.html (hunk 1 above)
Comment (Dariastep) [reply bookmarks.html:17]: Sure, thank you! I fixed it now.

**Pull request merged**